lib/phone_shop/screens/home_screen.dart

```diff
@@ -0,0 +1,158 @@
+  const HomeScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: const Color(0xFFF8F8F8),
+      appBar: _HomeScreenTitle(),
+      body: SafeArea(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: [
+            _ShopCardList(),
+            _DisabledButton(),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class _HomeScreenTitle extends StatelessWidget implements PreferredSizeWidget {
+  @override
+  Size get preferredSize => const Size.fromHeight(kToolbarHeight);
+
+  @override
+  Widget build(BuildContext context) {
+    return PreferredSize(
+      preferredSize: const Size.fromHeight(102),
+      child: AppBar(
+        backgroundColor: const Color(0xFFFFFFFF),
+        shadowColor: const Color.fromRGBO(0, 0, 0, 0.05),
+        shape: const RoundedRectangleBorder(
+          borderRadius: BorderRadius.only(
+            bottomLeft: Radius.circular(12),
+            bottomRight: Radius.circular(12),
+          ),
+        ),
+        leading: Transform.scale(scale: 0.9, child: _TitleIcon()),
+        titleSpacing: 8,
+        centerTitle: false,
+        title: Padding(
+          padding: const EdgeInsets.symmetric(vertical: 20),
+          child: _TitleText(),
+        ),
+      ),
+    );
+  }
+}
+
+class _TitleIcon extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(left: 20),
+      child: SvgPicture.asset(
+        'assets/images/Group.svg',
+        fit: BoxFit.contain,
+        width: 32,
+        height: 28.196,
+      ),
+    );
+  }
+}
+
+class _TitleText extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return ShaderMask(
+      shaderCallback: (Rect bounds) {
+        return const LinearGradient(
+          begin: Alignment.centerLeft,
+          end: Alignment.centerRight,
+          colors: [
+            Color(0xFF2AE0B1),
+            Color(0xFF048FF1),
+          ],
+        ).createShader(bounds);
+      },
+      child: RichText(
+        text: const TextSpan(
+          children: [
+            TextSpan(
+              text: 'Tech',
+              style: TextStyle(
+                fontFamily: 'Ubuntu',
+                fontSize: 23.327,
+                fontStyle: FontStyle.normal,
+                fontWeight: FontWeight.w400,
+              ),
+            ),
+            TextSpan(
+              text: 'Byte',
+              style: TextStyle(
+                fontFamily: 'Ubuntu',
+                fontSize: 23.327,
+                fontStyle: FontStyle.normal,
+                fontWeight: FontWeight.w700,
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class _ShopCardList extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Expanded(
+      child: Padding(
```

Comment:
Padding is not good, while scrolling it cuts below AppBar
<img width="425" alt="image" src="https://github.com/damjan-Igrowth/flutter-internship/assets/119666343/3139b2fb-9767-42a4-9f28-c849384d8981">